src/allmydata/test/test_storage_https.py

```diff
@@ -202,10 +206,6 @@ async def test_server_certificate_not_valid_yet(self):
             response = await self.request(url, certificate)
             self.assertEqual(await response.content(), b"YOYODYNE")
 
-        # We keep getting TLSMemoryBIOProtocol being left around, so try harder
-        # to wait for it to finish.
-        await deferLater(reactor, 0.001)
-
```

Comment:
Hrm. :/  I can't say that I can make much sense of this code motion.

Reply:
It should probably just go away given the other branch.